### DISCUSSION TITLE
Domains

### DISCUSSION THREAD

**Discussion by ghost** [community/community#37749] [Other Feature Feedback, Questions, & Ideas]
I am having an issue. I had a repository with a domain and I was given a certificate. I then deleted the repository, and I tried to add the domain to another repository, and it will not allow me to. It says: "Certificate already exists for aeecea.com (my domain) and is usable." So, my question is, how do I use it??? I am using GitHub pages with a custom domain to add on.

**Comment by jfullstackdev** [comment]
yup that's quite tricky, like deleting a repo then creating another repo with the same name and that will not be available,

maybe Support can help

https://support.github.com/contact/account

**Reply by ghost** [reply]
I have reached out, I will not mark this thread as solved until they're able to give me an answer, just in case someone else may know something. Thanks for your help though.
